### PR TITLE
get rid of distracting active checkboxes on empty stage page

### PR DESCRIPTION
**Pull request by grosser**
I always do a double-check on these ... would be easier to read if they were off by default
also adding an error when leaving all options blank for the slack webhook

screen shot 2017-03-09 at 7 34 20 pm
screen shot 2017-03-09 at 7 34 28 pm
screen shot 2017-03-09 at 8 37 24 pm

### CODE REVIEW

**Comment by grosser** [comment]
@sandlerr

File: plugins/slack_webhooks/app/models/slack_webhook.rb [hunk 1]
```diff
@@ -11,19 +12,23 @@ def deliver_for?(deploy_phase, deploy)
...
       URI.parse(webhook_url)
     rescue URI::InvalidURIError
       false
     end
 
     errors.add(:webhook_url, "is invalid") unless valid
   end
+
+  def validate_used
+    errors.add :base, "Pick a deliery option" if !for_buddy && !before_deploy && !after_deploy
```
Comment (sandlerr) [inline plugins/slack_webhooks/app/models/slack_webhook.rb]: delivery missing v

Comment (sandlerr) [inline plugins/slack_webhooks/app/models/slack_webhook.rb]: maybe easier to understand "select at least one delivery time" or something?

File: plugins/flowdock/db/migrate/20170310033621_remove_enabled_flag.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+class RemoveEnabledFlag < ActiveRecord::Migration[5.0]
+  class FlowdockFlow < ActiveRecord::Base
+  end
+
+  def up
+    FlowdockFlow.where(enabled: false).each do |flow|
+      puts "Deleting flow from stage #{flow.stage_id}: #{flow.name} #{flow.token}"
```
Comment (sandlerr) [inline plugins/flowdock/db/migrate/20170310033621_remove_enabled_flag.rb]: if this was being done by samson, it'd leak the tokens to whoever could see the deploy log :)

Comment (grosser) [inline plugins/flowdock/db/migrate/20170310033621_remove_enabled_flag.rb]: they were previously visible in plain sight anyway ... so not too worried ... for more sensitive things we could write them to cache and tell the user where they are ...

File: plugins/slack_webhooks/app/models/slack_webhook.rb (hunk 1 above)
Comment (grosser) [inline plugins/slack_webhooks/app/models/slack_webhook.rb]: fixed

**Pull request merged**